addon/webextension/selector/ui.js

```diff
@@ -420,6 +420,13 @@ this.ui = (function() { // eslint-disable-line no-unused-vars
       } else {
         this.el.classList.remove("bottom-selection");
       }
+
+      if (pos.right < 200 ) {
+        this.el.classList.add("left-selection");
+      } else {
+        this.el.classList.remove("left-selection")
```

Comment:
Not sure if this repo enforces it via ESLint, but this is "missing" a semicolon.